src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskGrantAccessMethod.cs

```diff
@@ -155,28 +155,23 @@ public override void ExecuteCmdlet()
             ParameterSetName = "DefaultParameter",
             Position = 1,
             Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            ValueFromPipeline = false)]
-        [AllowNull]
+            ValueFromPipelineByPropertyName = true)]
         [ResourceManager.Common.ArgumentCompleters.ResourceGroupCompleter()]
         public string ResourceGroupName { get; set; }
 
         [Parameter(
             ParameterSetName = "DefaultParameter",
             Position = 2,
             Mandatory = true,
-            ValueFromPipelineByPropertyName = true,
-            ValueFromPipeline = false)]
+            ValueFromPipelineByPropertyName = true)]
         [Alias("Name")]
-        [AllowNull]
         public string DiskName { get; set; }
 
-
         [Parameter(
             ParameterSetName = "DefaultParameter",
             Position = 3,
             Mandatory = true)]
-        public AccessLevel Access { get; set; }
+        public string Access { get; set; }
```

Comment:
@hyonholee same comment here, why the switch from this enum to string?

Reply:
@cormacpayne  This is changed in Swagger spec and client library.  They need string for Access because they will keep adding more access type, like VM sizes.